src/GameLogic/ItemPowerUpFactory.cs

```diff
@@ -308,25 +307,36 @@ private IEnumerable<PowerUpWrapper> CreateExcellentAndAncientBasePowerUpWrappers
 
         if (item.IsWizardryWeapon(out var staffRise))
         {
-            var additionalRise = ((int)staffRise * 25 / baseDropLevel) + 5;
+            var additionalRise = (((int)staffRise * 2 * 25 / baseDropLevel) + 5) / 2;
             yield return new PowerUpWrapper(new SimpleElement(additionalRise, AggregateType.AddRaw), Stats.StaffRise, attributeHolder);
             if (itemIsAncient)
             {
-                var ancientRiseBonus = 5 + (ancientDropLevel / 60);
+                var ancientRiseBonus = (2 + (ancientDropLevel / 60)) / 2;
                 yield return new PowerUpWrapper(new SimpleElement(ancientRiseBonus, AggregateType.AddRaw), Stats.StaffRise, attributeHolder);
             }
         }
 
         if (item.IsScepter(out var scepterRise))
         {
-            var additionalRise = ((int)scepterRise * 25 / baseDropLevel) + 5;
+            var additionalRise = (((int)scepterRise * 2 * 25 / baseDropLevel) + 5) / 2;
             yield return new PowerUpWrapper(new SimpleElement(additionalRise, AggregateType.AddRaw), Stats.ScepterRise, attributeHolder);
             if (itemIsAncient)
             {
-                var ancientRiseBonus = 5 + (ancientDropLevel / 60);
+                var ancientRiseBonus = (2 + (ancientDropLevel / 60)) / 2;
```

Comment:
https://github.com/kyleruss/emu-server/blob/202856a74c905c203b9b2795fd161f564ca8b257/GameServer/common/zzzitem.cpp#L702
Originally this is the magic power bonus, hence dividing by 2 at the end.